src/tokenizer.ts

```
@@ -0,0 +1,585 @@
+ */
+export function transitionFromInterpolationToLiteralMode(
+  state: InterpolationMode,
+  tokens: Token[],
+): void {
+  flushBuffer(state, tokens, TokenType.Interpolation);
+
+  const castedState = state as unknown as LiteralMode;
+  castedState.type = Mode.Literal;
+  castedState.buffer = "";
+
+  delete (state as AnyMode).n;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)